### PR TITLE
[CSR-4920] Basic table variant

### PR DESCRIPTION
**Pull request by dylanmau**
- Adds new table variant and adds some styling fixes to basic table and movable table. 
- Adds widget component 

After this is merged. The widget wrapper and the new table variant can be used together to create a component inside next and customer portals with a table inside a widget. 
Also the DashboardWidgetWrapper in next can be replaced by the tacugama component

### CODE REVIEW

**Review by tarashaidarTimechimp — APPROVED** [review]

File: src/components/widget-container/WidgetWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+  } = useTheme();
+
+  return (
+    <Box>
+      <Block
+        {...padding(scale0, scale800)}
+        display="flex"
+        justifyContent="space-between"
+        alignItems="center"
+        style={{ ...borderBottom({ ...border100, borderColor: light6 }) }}
```
Comment (hiddeottens) [inline src/components/widget-container/WidgetWrapper.tsx:27]: I would prefer not to use inline styles. Could this be wrapped with a `themedStyled()` func ?

File: src/components/basic-table/SecondaryBasicTable.tsx [hunk 2]
```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { TableBuilderColumn } from 'baseui/table-semantic';
+import { BasicTableProps, BasicTableRow, BasicTableColumnType } from './types';
+import { renderCell, BasicTableBuilder, BasicTableHeadCell } from './components';
+import { useBasicTableStyles } from './hooks';
+
+export const SecondaryBasicTable = ({ columns, emptyMessage, ...props }: BasicTableProps) => {
```
Comment (hiddeottens) [inline src/components/basic-table/SecondaryBasicTable.tsx:7]: I would prefer another name that more clearly indicates the difference between the two table types

**Review by hiddeottens — APPROVED** [review]
Nice

File: src/components/widget-container/WidgetWrapper.tsx (hunk 1 above)
Comment (dylanmau) [reply src/components/widget-container/WidgetWrapper.tsx]: Yes, you're right. changed it.

**Pull request merged**